spring-cloud-gcp-samples/spring-cloud-gcp-metrics-sample/pom.xml

```diff
@@ -28,6 +28,10 @@
     </dependencyManagement>
 
     <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-actuator</artifactId>
```

Comment:
@ttomsu When running from IntelliJ, removing it didn't break the sample for me, but now I see that it's actually required. In fact, I think it needs to be moved int our `spring-cloud-gcp-starter-metrics`.